crt/Makefile

```diff
@@ -24,10 +24,6 @@ ifdef MYST_ENABLE_GCOV
 DEFINES += -DMYST_ENABLE_GCOV
 endif
 
-ifdef MYST_PSEUDO_FORK
```

Comment:
We should remove `MYST_PSEUDO_FORK` altogether. Instead, default ForkMode to `none` if not specified.

Reply:
done